test/kubernetes/e2e/features/basicrouting/edge_suite.go

```diff
@@ -117,5 +123,51 @@ func (s *edgeBasicRoutingSuite) TestBasicVirtualServiceRouting() {
 			StatusCode: http.StatusOK,
 			Body:       gomega.ContainSubstring(testdefaults.NginxResponse),
 		})
+}
+
+func (s *edgeBasicRoutingSuite) TestVirtualServiceWithRetries() {
+	s.T().Cleanup(func() {
+		err := s.testInstallation.Actions.Kubectl().Delete(s.ctx, VirtualServiceWithRetriesYaml, "-n", s.testInstallation.Metadata.InstallNamespace)
+		s.Assert().NoError(err)
+	})
+
+	err := s.testInstallation.Actions.Kubectl().Apply(s.ctx, VirtualServiceWithRetriesYaml, "-n", s.testInstallation.Metadata.InstallNamespace)
+	s.Assert().NoError(err)
```

Comment:
Should we be asserting that the VS is up and running with something like lines 98-103?

Reply:
After reading further, I guess we're not using it directly so perhaps it doesn't matter.